lib/image-diff.js

```diff
@@ -8,25 +9,25 @@ var mkdirp = require('mkdirp');
 var tmp = require('tmp');
 
 // Define custom resize function
-// Taken from https://github.com/twolfson/twolfson.com/blob/3.4.0/test/perceptual-tests/twolfson.com_test.js#L88-L107
-// TODO: Make image resizing its own library
-// DEV: This does not pollute gm.prototype
-gm.prototype.fillFromTo = function (params) {
+function transparentExtent(gm, params) {
+  // Assert we received our parameters
+  assert.notEqual(params.width, undefined);
+  assert.notEqual(params.height, undefined);
```

Comment:
Why not more readable error messages? These parameters are read in using input from the user (eventually) so there's a chance the consuming code could have erroneous input. An assert doesn't really explain the issue all that well.